### DISCUSSION TITLE
Peer List as Command Line Argument

### DISCUSSION THREAD

**Discussion by ejMina226** [MinaProtocol/mina#14766] [Ideas]
Peer list URL should usually be None. This option is better provided with a command line argument. Putting it in the config makes the network explicitly rely on a certain number of nodes, reducing decentralisation

**Comment by emberian** [comment]
Currently the daemon supports:

- `--peer` 
- `--peer-list-file`
- `--peer-list-url`
- `--direct-peer`

as ways to configure its connection set, with no defaults. Can you clarify what should be different?
